### PR TITLE
Timeout and new directions

### PR DESCRIPTION
**Pull request by andreeaceachir142**

### CODE REVIEW

File: packages/map-template/src/components/Directions/Directions.jsx [hunk 1]
```diff
@@ -4,26 +4,22 @@ import { useTranslation } from 'react-i18next';
 import { useRecoilState, useRecoilValue } from 'recoil';
 import mapsIndoorsInstanceState from '../../atoms/mapsIndoorsInstanceState';
 import travelModeState from '../../atoms/travelModeState';
-import { ReactComponent as CloseIcon } from '../../assets/close.svg';
-import { ReactComponent as ClockIcon } from '../../assets/clock.svg';
-import { ReactComponent as WalkingIcon } from '../../assets/walk.svg';
-import { ReactComponent as DriveIcon } from '../../assets/drive.svg';
-import { ReactComponent as BikeIcon } from '../../assets/bike.svg';
```
[the remark this answers is not in the document]
Comment (andreeaceachir142) [reply packages/map-template/src/components/Directions/Directions.jsx:11]: Yes, we don't use these anymore

[the remark this answers is not in the document]
Comment (andreeaceachir142) [reply packages/map-template/src/components/Directions/Directions.jsx:11]: They are used on the `Wayfinding.jsx` page though

[the remark this answers is not in the document]
Comment (ammapspeople) [reply packages/map-template/src/components/Directions/Directions.jsx:11]: Then they probably shouldn't 😄

[the remark this answers is not in the document]
Comment (andreeaceachir142) [reply packages/map-template/src/components/Directions/Directions.jsx:11]: No, no, they should be removed from this file. That is correct.

**Pull request merged**